computestorage/storage.go

```diff
@@ -7,7 +7,7 @@ import (
 	hcsschema "github.com/Microsoft/hcsshim/internal/hcs/schema2"
 )
 
-//go:generate go run ../mksyscall_windows.go -output zsyscall_windows.go storage.go
+//go:generate go run github.com/Microsoft/go-winio/tools/mkwinsyscall -output zsyscall_windows.go storage.go
```

Comment:
I didnt even know you could do a `go run` from a remote. Thats crazy

Reply:
I think if you try and use `$GOROOT/src/syscall/mksyscall_windows.go`, it tells you the prefered way `golang.org/x/sys/windows/mkwinsyscall`

Reply:
Also did not know..

Reply:
Would this actually work for a vendor based project (one that had winio vendored)? I did a quick test of just vendoring x/sys/windows and doing the go:generate dance and specifying golang.org/x/sys/windows/mkwinsyscall, it complains with: 
cannot find package "github.com/dcantah/generate/vendor/golang.org/x/sys/windows/mkwinsyscall" in:
        C:\Users\danny\go\src\github.com\dcantah\generate\vendor\golang.org\x\sys\windows\mkwinsyscall

which is honestly what I thought would happen, but maybe there's more I'm missing. I don't think we'd ever actually have an import from the tools/mkwinsyscall pkg so mod vendor wouldn't pull in the files afaict.

Reply:
So there is a way to vendor the tool:
https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module
I have to wait until upstream go-winio gets updated, since it fails to vendor and update.
I am not sure if Go respects that local copy when building it though...

Reply:
Worth a shot, it sounds like there is a way though